crates/win-api-wrappers/src/netmgmt.rs

```diff
@@ -1,49 +1,30 @@
-use anyhow::bail;
+use anyhow::{bail, Context as _};
 use windows::Win32::Foundation::WIN32_ERROR;
-
-use crate::identity::sid::{RawSid, Sid};
-use crate::raw::Win32::NetworkManagement::NetManagement::{
-    NERR_Success, NetApiBufferFree, NetLocalGroupAddMembers, NetLocalGroupDelMembers, NetLocalGroupGetMembers,
-    LOCALGROUP_MEMBERS_INFO_0, MAX_PREFERRED_LENGTH,
+use windows::Win32::NetworkManagement::NetManagement;
+use windows::Win32::NetworkManagement::NetManagement::{
+    NetLocalGroupGetMembers, LOCALGROUP_MEMBERS_INFO_0, MAX_PREFERRED_LENGTH,
 };
-use crate::raw::Win32::Security::WinBuiltinAdministratorsSid;
-use crate::utils::WideString;
+use windows::Win32::Security::WinBuiltinAdministratorsSid;
+
+use crate::identity::sid::Sid;
+use crate::str::{U16CStr, U16CStrExt as _, U16CString};
 use crate::Error;
 
 pub fn add_local_group_member(group_name: &str, security_identifier: &Sid) -> anyhow::Result<()> {
-    // SAFETY: The structure is a simple POD type.
-    let mut group_info = unsafe { core::mem::zeroed::<LOCALGROUP_MEMBERS_INFO_0>() };
-
-    let group_name = WideString::from(group_name);
-
-    let user_sid = RawSid::try_from(security_identifier)?;
-    group_info.lgrmi0_sid = user_sid.as_psid();
-
-    // SAFETY: group_name holds a null-terminated UTF-16 string, and as_pcwstr() returns a valid pointer to it.
-    let rc =
-        unsafe { NetLocalGroupAddMembers(None, group_name.as_pcwstr(), 0, &group_info as *const _ as *const u8, 1) };
-
-    if rc != NERR_Success {
-        bail!(Error::from_win32(WIN32_ERROR(rc)))
-    }
+    let group_name = U16CString::from_str(group_name).context("invalid group name")?;
 
-    Ok(())
-}
-
-pub fn remove_local_group_member(group_name: &str, security_identifier: &Sid) -> anyhow::Result<()> {
```

Comment:
Why remove this function? It's not currently used, but we would like to use that in future

If it's the case that we want to strip unused code for now; this whole module is technically not used by the application currently